save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/FiltersRow.kt

```diff
@@ -19,86 +21,144 @@ import react.dom.html.ReactHTML.option
+                div {
+                    className = ClassName("col-auto align-self-center")
+                    +"File name: "
+                }
+                div {
+                    className = ClassName("col-auto")
+                    input {
+                        type = InputType.text
+                        className = ClassName("form-control")
+                        value = filters.fileName
+                        required = false
+                        onChange = {
+                            setFilters(filters.copy(fileName = it.target.value))
+                        }
                     }
                 }
             }
             div {
-                className = ClassName("col-auto align-self-center")
-                +"Test suite: "
+                className = ClassName("row")
+                div {
+                    className = ClassName("col-auto align-self-center")
+                    +"Test suite: "
+                }
+                div {
+                    className = ClassName("col-auto")
+                    input {
+                        type = InputType.text
+                        className = ClassName("form-control")
+                        value = filters.testSuite
+                        required = false
+                        onChange = {
+                            setFilters(filters.copy(testSuite = it.target.value))
+                        }
+                    }
+                }
             }
             div {
-                className = ClassName("col-auto")
-                input {
-                    type = InputType.text
-                    className = ClassName("form-control")
-                    defaultValue = initialValueTestSuite
-                    required = false
-                    onChange = {
-                        setTestSuite(it.target.value)
+                className = ClassName("row")
+                div {
+                    className = ClassName("col-auto align-self-center")
+                    +"Tags: "
+                }
+                div {
+                    className = ClassName("col-auto")
+                    input {
+                        type = InputType.text
+                        className = ClassName("form-control")
+                        value = filters.tag
+                        required = false
+                        onChange = {
+                            setFilters(filters.copy(tag = it.target.value))
+                        }
                     }
                 }
             }
             button {
                 className = ClassName("btn btn-primary")
                 fontAwesomeIcon(icon = faSearch, classes = "trash-alt")
-                // +"Find"
                 onClick = {
-                    onChangeStatus(status)
-                    onChangeTestSuite(testSuite)
+                    props.onChangeFilters(Filters(filters.status, filters.fileName, filters.testSuite, filters.tag))
```

Comment:
Seems like you need to invoke smth line `onFindButtonPressed` because filters change higher

Reply:
Done